src/echo.js

```diff
@@ -23,12 +27,23 @@ function _echo(opts, messages) {
   // allow strings starting with '-', see issue #20
   messages = [].slice.call(arguments, opts ? 0 : 1);
 
-  if (messages[0] === '-e') {
-    // ignore -e
+  var option = ['-e', '-n', '-ne', '-en'].indexOf(messages[0]);
+  var output;
+
+  if (option >= 0) {
```

Comment:
Ok, I've added a function to `echo.js` that parses options just for `echo`. This means adding `-E` later on will be easier.